spaces/S000024/properties/P000187.md

```diff
@@ -0,0 +1,7 @@
+---
+space: S000024
+property: P000187
+value: true
+---
+
+If $x\in \mathbb{R}$ then player 1 can choose $U_n = \{x\}$, then necessarily $p_n = x$. If $x = \infty_i$ then player 1 can choose $U_n = \{\infty_i\}\cup X\setminus \{p_1, ..., p_{n-1}\}$. Since any sequence of distinct elements of $X$ converges to $\infty_i$, it follows that $p_n$ converges to $\infty_i$. 
```

Comment:
```suggestion
If $x\in \mathbb{R}$ then player 1 can choose $U_n = \{x\}$; then necessarily $p_n = x$. If $x = \infty_i$ then player 1 can choose $U_n = \{\infty_i\}\cup(X\setminus \{p_1, ..., p_{n-1}\})$. Since any sequence of distinct elements of $X$ converges to $\infty_i$, it follows that $p_n$ converges to $\infty_i$. 
```